varats/data/reports/blame_report.py

```diff
@@ -105,6 +150,39 @@ def __str__(self) -> str:
         return str_representation
 
 
+def _calc_diff_between_func_entries(
+        base_func_entry: BlameResultFunctionEntry,
+        prev_func_entry: BlameResultFunctionEntry) -> BlameResultFunctionEntry:
+    diff_interactions: tp.List[BlameInstInteractions] = []
+
+    base_interactions = sorted(base_func_entry.interactions)
+    prev_interactions = sorted(prev_func_entry.interactions)
+
+    for base_inter in base_interactions:
+        if base_inter in prev_interactions:
+            prev_inter_idx = prev_interactions.index(base_inter)
+            prev_inter = prev_interactions[prev_inter_idx]
```

Comment:
```suggestion
            prev_inter = prev_interactions.pop(prev_inter_idx)
```
And remove the `del` below.